### DISCUSSION TITLE
🙏 Help: How to trigger blur event on kol-buttons?

### DISCUSSION THREAD

**Discussion by mukki182** [public-ui/kolibri#6795] [Q&A]
### Summary

on mobile devices chrome will keep the hover effect on button click until something else is pressed.
normally i would reset this with a blur event, but this way doesn't work:
```
button._on = {
    onClick: (event) => {
        ... do stuff
        event.target.blur();
    }
};
```
how do you do this with kol-buttons?

**Comment by deleonio** [comment]
We have some feature requests for this: https://github.com/public-ui/kolibri/issues/6343

Thanks for your feedback ... so we can handle the priority.
